### DISCUSSION TITLE
"Show Browser Manager Sidebar" button for the toolbar?

### DISCUSSION THREAD

**Discussion by GollyJer** [Floorp-Projects/Floorp#1464] [Feature Request]
The available "Show Sidebars" button is for the traditional firefox sidebar. 
I was hoping to find one of similar functionality but for the Browser Manager sidebar. 
Per the screenshot below, it looks like maybe on is supposed to be available, but it actually opens the bookmark manager.

<img width=360 src="https://github.com/user-attachments/assets/f80ab668-411f-4bcf-b663-e445c0d6b58e" />

**Comment by CutterKnife** [comment]
There is no button to do that, but you can toggle BMS with the Keyboard Shortcut setting “floorp-show-bsm” (failing localization).
